libtuf/src/main/scala/com/advancedtelematic/libtuf/crypt/TufCrypto.scala

```diff
@@ -38,7 +39,13 @@ trait TufCrypto[T <: KeyType] {
 
   def encode(keyVal: T#Priv): Json
 
-  def generateKeyPair(keySize: Int): TufKeyPair
+  def generateKeyPair(keySize: Int): TufKeyPair = {
+    if(!validKeySize(keySize))
```

Comment:
Probably `require(validKeySize(keySize), "Key size is too small")` would be better